src/collector.ts

```diff
@@ -161,7 +161,11 @@ class GomodDependencyCollector implements IDependencyCollector {
             exec(`${config.golang_executable} list -f '{{ join .Imports "\\n" }}' ./...`,
                    { cwd: vscodeRootpath, maxBuffer: 1024 * 1200 }, (error, stdout, stderr) => {
                 if (error) {
-                    reject(`'${config.golang_executable} list' command failed with error :: ${stderr}`);
+                    if (error.code == 127) { // Invalid command, go executable not found
+                        reject(`Unable to locate '${config.golang_executable}' executable, Restart all instances of visual code editor and try again`)
```

Comment:
Error statement is not localised. It will look weird on IntelliJ.

Also semi colon is missing.

```suggestion
                        reject(`Unable to locate '${config.golang_executable}' executable`);
```

Reply:
Should I replace 'visual code editor' to 'code editor' or 'editor'?

Reply:
Resolved this as per slack discussion to have message as "Unable to locate '${goexecutable}'"